test/fixtures/election-98f5203139-choctaw-general-2019/election.json

```diff
@@ -5,7 +5,7 @@
     "id": "10",
     "name": "Choctaw County"
   },
-  "date": "November 5, 2019",
+  "date": "2019-11-05T00:00:00-05:00",
```

Comment:
This and the other changes in this file were because this `election.json` didn't parse correctly.